api/lib/validation/attributeValidator.js

```diff
@@ -33,6 +33,10 @@ var normalizeEndDate = function(object) {
   return object.end_date;
 }
 
+var isInt = function(n) {
+  return n % 1 === 0;
+}
+
```

Comment:
According to the sample in https://www.npmjs.com/package/jsonschema ,  the change in schemaValidator.js is enough.   Why we need to check attributeValidator again?